debian/control

```diff
@@ -0,0 +1,44 @@
+Source: yices2
+Maintainer: Ian A. Mason <iam@csl.sri.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9), dh-autoreconf (>= 9)
+Homepage: http://yices.csl.sri.com/
+
+
+Package: libyices2.6
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Pre-Depends: ${misc:Pre-Depends}
+Multi-Arch: same
+Description:  Yices SMT Solver API Library
+ Long description goes here.
+
+
+Package: yices2
+Architecture: any
+Depends:  gperf, libgmp-dev,  libpoly0 (>= 1.3), ${shlibs:Depends}, ${misc:Depends}
+Description: Yices SMT Solver
+ Long description goes here.
+
+Package: yices2-dev
+Section: libdevel
+Architecture: any
+Depends: yices2,  libyices2.6 (= ${binary:Version}), ${misc:Depends}, libgmp-dev
+Description: C library for manipulating multivariate polynomials
+ Used by the SMT solver yices2.
+ .
+ This package contains development headers for libpoly.
```

Comment:
Right. Though it might actually contain the symbols for libpoly, because I don't think
it contains those for yices yet :-)